### ISSUE TITLE
Relative links from GitHub readme

### ISSUE THREAD

**Issue by xen2** [daveaglick/discoverdotnet#234]
Relative links to git repo on README are broken.
Example: https://discoverdot.net/projects/xenko
Any of the three links in `License` category are broken:
*MIT* should point to https://github.com/xenko3d/xenko/blob/master/LICENSE.md rather than https://discoverdot.net/projects/LICENSE.md

**Comment by daveaglick** [comment]
Good catch - right now it just renders whatever Markdown is in the readme as-is. I should probably rewrite relatives URLs or set a base path for the readme portion.
